wai-handler-hal.cabal

```diff
@@ -48,15 +47,15 @@ common opts
 
 common deps
   build-depends:
-    , base                  >=4.12      && <4.19
+    , base                  >=4.12      && <4.21
```

Comment:
This upper bound admits GHC 9.8 and 9.10. Can you update `tested-with` to swap GHC 9.6.2 with 9.6.5 and add entries for 9.8.2 and 9.10.1?

Reply:
And once that's done, do another `haskell-ci regenerate`.

Reply:
Done.